compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/platform/ApplicationAccessibilityTest.kt

```diff
@@ -0,0 +1,306 @@
+                Column {
+                    Button(
+                        onClick = {},
+                        modifier = Modifier.size(20.dp),
+                    ) {
+                        Text("Accessible button 1")
+                    }
+                    Button(
+                        onClick = {},
+                        modifier = Modifier.size(20.dp),
+                    ) {
+                        Text("Accessible button 2")
+                    }
+                }
+            }
+        }
+        awaitIdle()
+
+        checkAccessibleOnPoint(window, 10, 10) {
+            assertThat(accessibleName).isEqualTo("Accessible button 1")
+        }
+
+        checkAccessibleOnPoint(window, 5, 22) {
+            assertThat(accessibleName).isEqualTo("Accessible button 2")
+        }
+    }
+
+    // TODO: component under popup shouldn't be read by screen reader
+    //  but current implementation does it
+    //  (see ComposeSceneAccessible.ComposeSceneAccessibleContext.getAccessibleAt)
+    @Ignore
+    @Test
+    fun `hover popup when there is a component under it`() = runApplicationTest {
+        lateinit var window: ComposeWindow
+
+        launchTestApplication {
+            Window(onCloseRequest = {}) {
+                window = this.window
+                Column {
+                    Button(
+                        onClick = {},
+                        modifier = Modifier.size(20.dp),
+                    ) {
+                        Text("button under popup")
+                    }
+                    val popupPosition = object : PopupPositionProvider {
+                        override fun calculatePosition(
+                            anchorBounds: IntRect,
+                            windowSize: IntSize,
+                            layoutDirection: LayoutDirection,
+                            popupContentSize: IntSize
+                        ): IntOffset = IntOffset.Zero
+                    }
+                    Popup(popupPosition) {
+                        Column {
+                            Spacer(Modifier.height(30.dp))
+                            Button(
+                                onClick = {},
+                                modifier = Modifier.size(20.dp)
+                            ) {
+                                Text("popup button")
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        awaitIdle()
+
+        checkAccessibleOnPoint(window, 5, 32) {
+            assertThat(accessibleName).isEqualTo("popup button")
+        }
+
+        checkAccessibleOnPoint(window, 5, 5) {
+            assertNotEquals("button under popup", accessibleName)
+        }
+    }
+
+    // https://github.com/JetBrains/compose-multiplatform/issues/2185
+    @Test
+    fun `drop-down menu accessibility`() = runApplicationTest {
+        lateinit var window: ComposeWindow
+        var firstItemPositionPx: Offset? = null
```

Comment:
Could be `lateinit var`

Reply:
`'lateinit' modifier is not allowed on local variables of inline class types.`